bin/ch_fuse.c

```diff
@@ -119,7 +119,7 @@ void sq_fork(struct container *c)
       T_ (asprintf(&subdir, "/%s.ch/mnt", username) > 0);
       c->newroot = cat("/var/tmp", subdir);
       INFO("using default mount point: %s", c->newroot);
-      mkdirs("/var/tmp", subdir, NULL, 0);
+      mkdirs("/var/tmp", subdir, NULL);
```

Comment:
Seems like the other files have more `NULL`s converted to `0` this PR. Why is this one opposite?

Reply:
This one was and is a null pointer. The zero that was removed was the count of items.